packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -247,12 +238,12 @@ const isDoughnutChart = computed<boolean>(() => props.chartOptions.type === Char
 const barChartOrientation = computed<'horizontal' | 'vertical'>(() => props.chartOptions.type.includes('Vertical') ? 'vertical' : 'horizontal')
 
 const metricAxesTitle = computed<string | undefined>(() => {
-  if (!props.chartData?.meta.metricNames || !props.chartData?.meta.metricUnits) {
+  if (!props.chartData?.meta.metric_names || !props.chartData?.meta.metric_units) {
```

Comment:
Minor: duplicate conditional.

Reply:
one is checking for `metric_names` and the other is checking for `metric_units`

Reply:
Aaah, sorry, my bad.